src/app/views/title/index.ejs

```diff
@@ -3,63 +3,107 @@
 <div class="ui main text container" id="container">
     <div class="ui top attached">
 
-        <div class="ui search focus">
+        <div class="ui horizontal segments grid">
+          <div class="ui segment ten wide column">
+            <div class="ui search focus">
             <div class="ui icon input">
                 <input class="search" id="newTitle" type="text" placeholder="search to add..." name="title[name]" required autocomplete="off" onblur="hideSuggestions()">
                 <i class="search icon"></i>
             </div>
             <div class="results" id="results"></div>
-        </div>
-
-        <div class="ui list">
-        <% if(titles.length == 0) { %>
-            <div class="item title">
-                <br>
-                Seems like you haven't added any titles to your watchlist yet.
-                <br>
-                Start by typing in the search field above.
-                <br>
-                .
             </div>
-        <% } %>
 
-        <% titles.forEach(function(title){ %>          
-            <div class="item title">
-                <img class="ui image" src="<%= title.poster %>" height="150px" width="100px">
-                <div class="content">
-                    <span id="titleName" class=" <% if(title.seen) { %> seen  <% } else { %> unseen <% } %> "><%= title.name %></span>
-                    <div class="description">
-                        (<%= title.year %>)
+            <div class="ui list">
+            <% if(titles.filter(title=>!title.seen).length == 0) { %>
+                <div class="item title">
+                    <h1>😭</h1> Seems like you haven't added any titles to your watchlist yet or your list is empty.
+                    <br>
+                    Start by typing in the search field above.
+                </div>
+            <% } %>
+
+            <% titles.forEach(function(title){ %>
+                <% if(!title.seen) { %>
```

Comment:
the <% %> tags are not required for each and every line of javascript but everything between them is considered javascript

Reply:
ok good to know, but I think the readability is better with it :)